hs_tracking/management/commands/stats.py

```diff
@@ -0,0 +1,222 @@
+            help="user stats by month",
+        ),
+        make_option(
+            "--monthly-orgs-counts",
+            dest="monthly_orgs_counts",
+            action="store_true",
+            help="unique organization stats by month",
+        ),
+        make_option(
+            "--users-details",
+            dest="users_details",
+            action="store_true",
+            help="current user list",
+        ),
+        make_option(
+            "--resources-details",
+            dest="resources_details",
+            action="store_true",
+            help="current resource list with sizes",
+        ),
+        make_option(
+            "--monthly-users-by-type",
+            dest="monthly_users_by_type",
+            action="store_true",
+            help="user type stats by month",
+        ),
+        make_option(
+            "--yesterdays-variables",
+            dest="yesterdays_variables",
+            action="store_true",
+            help="dump tracking variables collected today",
+        ),
+    )
+
+    def print_var(self, var_name, value, period=None):
+        timestamp = timezone.now()
+        if not period:
+            print("{}: {} {}".format(timestamp, var_name, value))
+        else:
+            start, end = period
+            print("{}: ({}/{}--{}/{}) {} {}".format(timestamp,
+                                                    start.year, start.month,
+                                                    end.year, end.month,
+                                                    var_name, value))
+
+    def monthly_users_counts(self, start_date, end_date):
+        profiles = User.objects.filter(date_joined__lte=end_date, is_active=True)
+        self.print_var("monthly_users_counts", profiles.count(),
+                       (start_date, end_date))
+
+    def monthly_orgs_counts(self, start_date, end_date):
+        profiles = UserProfile.objects.filter(user__date_joined__lte=end_date)
+        org_count = profiles.values('organization').distinct().count()
+        self.print_var("monthly_orgs_counts", org_count, (start_date, end_date))
+
+    def monthly_users_by_type(self, start_date, end_date):
+        user_types = UserProfile.objects.values('user_type').distinct()
+        for ut in [_['user_type'] for _ in user_types]:
+            ut_users = User.objects.filter(userprofile__user_type=ut)
+            sessions = hs_tracking.Session.objects.filter(
+                Q(begin__gte=start_date) &
+                Q(begin__lte=end_date) &
+                Q(visitor__user__in=ut_users)
+            )
+            self.print_var("active_{}".format(ut),
+                           sessions.count(), (end_date, start_date))
+
+    def users_details(self):
+        w = csv.writer(sys.stdout)
+        fields = [
+            'first name',
+            'last name',
+            'email',
+            'user type',
+            'organization',
+            'created date',
+            'last login',
+        ]
```

Comment:
Rearrange these columns so that user `created date` is first to be consistent with typical logging format.  This also makes the logs more human readable.